audioswitch/src/test/java/com/twilio/audioswitch/BaseTest.kt

```diff
@@ -8,19 +8,16 @@ import android.content.Context
 import android.media.AudioManager
 import com.nhaarman.mockitokotlin2.mock
 import com.nhaarman.mockitokotlin2.whenever
-import com.twilio.audioswitch.DEVICE_NAME
 import com.twilio.audioswitch.android.BuildWrapper
 import com.twilio.audioswitch.android.LogWrapper
+import com.twilio.audioswitch.bluetooth.BluetoothHeadsetManager
 import com.twilio.audioswitch.selection.AudioDeviceChangeListener
 import com.twilio.audioswitch.selection.AudioDeviceManager
 import com.twilio.audioswitch.selection.AudioDeviceSelector
 import com.twilio.audioswitch.selection.AudioFocusRequestWrapper
-import com.twilio.audioswitch.setupAudioManagerMock
-import com.twilio.audioswitch.setupScoHandlerMock
-import com.twilio.audioswitch.setupSystemClockMock
 import com.twilio.audioswitch.wired.WiredHeadsetReceiver
 
-open class BaseTest {
+open class BaseTest() {
```

Comment:
I think we can get rid of the parenthesis here since there are no constructor parameters.